.npmrc

```diff
@@ -0,0 +1 @@
+@metrisk:registry=https://npm.pkg.github.com
```

Comment:
Should this not mimic the encryption lib? Think as it will now be under the Metrisk GitHub scope we should make it private too?

Reply:
Yeah if we have them under the @metrisk scope, it could be just seen as internal tooling so makes sense to be private. We should just need to change this to have the `GITHUB_TOKEN` and to remove the access public from the `package.json`